tests/test_is_satisfying_hint.py

```diff
@@ -398,3 +400,16 @@ def test_isattr():
     assert not isattr(IntAttr(1), BaseAttr(StringAttr))
     assert isattr(IntAttr(1), EqAttrConstraint(IntAttr(1)))
     assert not isattr(IntAttr(1), EqAttrConstraint(IntAttr(2)))
+
+
+################################################################################
+# SSAValue
+################################################################################
+
+
+def test_ssavalue():
+    a = TestSSAValue(i32)
+
+    assert isa(a, SSAValue)
+    assert isa(a, SSAValue[IntegerType])
+    assert not isa(a, SSAValue[StringAttr])
```

Comment:
Could we have a test with a generic attribute like `VectorType[IntegerType]` or something.